app/scripts/translations/en.js

```diff
@@ -77,7 +77,7 @@ NAV_InteractContract        : 'Interact with Contract ',
 NAV_Multisig                : 'Multisig ',
 NAV_MyWallets               : 'My Wallets ',
 NAV_Offline                 : 'Send Offline ',
-NAV_SendEther               : 'Send Ether & Tokens ',
+NAV_SendEther               : 'Send Ella & Tokens ',
```

Comment:
Right now can you revert this single line of change? We haven't decided the term yet.

Like Ethereum, we can call the money by the symbol: "You have 1.0 ELLA" or "You have 1.0 ETH", but for all currency terms like "Ether", "Finney", "Szabo", "Shannon" we share them with Ethereum.